rpm/qtbase.spec

```diff
@@ -825,10 +836,12 @@ install -D -p -m 0644 %{_sourcedir}/qt.conf %{buildroot}%{_libdir}/qt5/bin/qt.co
 %{_libdir}/pkgconfig/Qt5PrintSupport.pc
 %{_datadir}/qt5/mkspecs/modules/qt_lib_printsupport.pri
 %{_datadir}/qt5/mkspecs/modules/qt_lib_printsupport_private.pri
+%{_libdir}/cmake/Qt5PrintSupport
 
 %files qtconcurrent
 %defattr(-,root,root,-)
 %{_libdir}/libQt5Concurrent.so.*
+%{_libdir}/cmake/Qt5Concurrent
```

Comment:
Shouldn't this me in qtconcurrent-devel?

Reply:
Indeed.